internal/vulnerability/runner.go

```diff
@@ -33,7 +34,13 @@ type VulnerabilityRunner struct {
 }
 
 func NewVulnerabilityRunner(log *logp.Logger) (VulnerabilityRunner, error) {
+	ctx := context.Background() // TODO: carry
```

Comment:
Todo 😁

Reply:
Yeah, context.Background was used in this constructor function before these changes and thus the todo, because is out of the scope of this pr.

Reply:
ok removed the comment and I will open a task